pepper-apis/config/application.conf.ctmpl

```diff
@@ -67,6 +67,12 @@
         "scanResultSubscription": "{{$conf.Data.fileUploads.scanResultSubscription}}",
         "signerServiceAccount": {{$conf.Data.fileUploads.signerServiceAccount | toJSONPretty}}
     },
+    "fileDownload":{
+        "signerServiceAccount": {{$conf.Data.fileDownload.signerServiceAccount | toJSONPretty}}
+        "maxSignedUrlMins": 5,
+        "defaultDownloadBucket": "{{$conf.Data.fileDownload.defaultDownloadBucket}}"
```

Comment:
I've been thinking about this in my own code and have decided that I don't need this in configuration.  The flow I'm expecting in DSM is:
DSM authorizes upload to bucket `studyA-upload` (where studyA is the notional name of the study)
User uploads to studyA bucket via DSM UI (POST, non-resumable).

- Anti-virus notified about new upload.
- Upload moved to final resting bucket (e.g. `studyA`)
- Message posted to DSM with the original and new bucket/blob names and/or virus scanning outcome.
- DSM records new bucket/blob names and virus scanning outcome.

Only after DSM stores the scanning outcome will the Follow-Up message be possible to generate.  In that message that we pass to DSS will need to be the bucket name, blob name and ddp participant id.

Because the anti-virus is going to determine the final resting spot for the data, I got rid of my local configuration for the final resting spots based on realm.  They need to be in the messages and not in the configuration.  IMHO, any "default" handling should fail so that we don't accidentally return something we're unsure about.

I also realize that you might want this value in for testing.